target/linux/ath79/dts/qca9531_comfast_cf-ew72.dts

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-ew72", "qca,qca9531";
+	model = "COMFAST CF-EW72";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &wan;
+		led-failsafe = &wan;
+		led-upgrade = &wan;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins>;
+
+		lan: lan {
```

Comment:
Should be
`led_lan: lan {`

Please update below accordingly.

However, at the moment it does not look like you'd be using the lan label at all? In that case, remove it.

Which LED is used by the stock FW for signalling boot etc.?

Reply:
I'm honestly not sure, I'm not on the stock firmware anymore and that's one thing I didn't check prior to getting onto this version of OpenWRT.  To my knowledge it's the WAN.

Reply:
Confirmed on the factory firmware it is the WAN LED used for signaling purposes.